playground/src/experiments/experiments.scss

```diff
@@ -8,8 +8,8 @@
 
   .experiments-list {
     max-height: 70%;
-    padding: 0;
     overflow-y: scroll;
```

Comment:
Make this `overflow-y: auto` to prevent a scrollbar from showing when it is not needed.